gui.go

```diff
@@ -722,23 +733,29 @@ func (g *Gui) onKey(ev *termbox.Event) error {
 // and event. The value of matched is true if there is a match and no errors.
 func (g *Gui) execKeybindings(v *View, ev *termbox.Event) (matched bool, err error) {
 	var globalKb *keybinding
+
 	for _, kb := range g.keybindings {
 		if kb.handler == nil {
 			continue
 		}
+
 		if !kb.matchKeypress(Key(ev.Key), ev.Ch, Modifier(ev.Mod)) {
 			continue
 		}
+
 		if kb.matchView(v) {
 			return g.execKeybinding(v, kb)
 		}
-		if kb.viewName == "" && ((v != nil && !v.Editable) || kb.ch == 0) {
+
+		if kb.viewName == "" && ((v != nil && !v.Editable) || (kb.ch == 0 && kb.key != KeyCtrlU && kb.key != KeyCtrlA && kb.key != KeyCtrlE)) {
```

Comment:
this is kinda tricky: we're saying that if the keybinding is global, but the focused view is editable and you've pressed KeyCtrlU or KeyCtrlA, we don't want to execute that keybinding because we want the editor to receive the event.

If we wanted to remove those ctrl+A and ctrl+U keybindings from the simple editor, this logic is not what we want. But then let's say in lazygit we use an editor which does support those keys, if we have a conflicting global keybinding it will be executed which we don't want. It seems like we need some way of saying that if the focused view is editable, check if the key is handled by the editor, and if so, don't execute the global keybinding. I'm not sure how we would do that without changing the case statement in the editor to a map of keys to handler functions, and then accessing that map via the view at this line and seeing if the key is included in the map. What are your thoughts?

Reply:
The map could be a solution. I was thinking about an keybinding blacklist. We just let lazygit use a custom editor. Then lazygit would add ctrl+u and ctrl+a to a blacklist.

Reply:
What do you think about that @jesseduffield